index.js

```diff
@@ -4,7 +4,7 @@ var Dyno = require('dyno');
 var split = require('split');
 var Readable = require('stream').Readable;
 
-module.exports = function(method, database, migrate, stream, live, concurrency, callback) {
+module.exports = function(method, database, migrate, stream, live, json, concurrency, callback) {
```

Comment:
._.

yeah, but at least the user doesn't have to sort through constructing this function in the proper order.